tests/functional/utils/test_oracles.py

```diff
@@ -1,32 +1,56 @@
 import pytest
 
+from brownie import (
+    BTCOSMedianizer,
+    ETHOSMedianizer,
+)
 
-@pytest.fixture
-def oracle(a, BTCOSMedianizer):
-    yield a[0].deploy(BTCOSMedianizer)
+ORACLES = [BTCOSMedianizer, ETHOSMedianizer]
 
 
-def test_hardcoded_config(a, oracle):
-    assert oracle.OSM() == "0xf185d0682d50819263941e5f4EacC763CC5C6C42"
-    assert oracle.MEDIANIZER() == "0x9B8Eb8b3d6e2e0Db36F41455185FEF7049a35CaE"
+@pytest.mark.parametrize(
+    "Oracle,osm,medianizer",
+    [
+        (
+            BTCOSMedianizer,
+            "0xf185d0682d50819263941e5f4EacC763CC5C6C42",
+            "0x9B8Eb8b3d6e2e0Db36F41455185FEF7049a35CaE",
+        ),
+        (
+            ETHOSMedianizer,
+            "0x81FE72B5A8d1A857d176C3E7d5Bd2679A9B85763",
+            "0x729D19f657BD0614b4985Cf1D82531c67569197B",
+        ),
+    ],
+)
+def test_hardcoded_config(a, Oracle, osm, medianizer):
+    oracle = a[0].deploy(Oracle)
+    assert oracle.OSM() == osm
+    assert oracle.MEDIANIZER() == medianizer
 
 
-def test_governance(a, gov, oracle):
+@pytest.mark.parametrize("Oracle", ORACLES)
+def test_governance(a, gov, Oracle):
+    oracle = a[0].deploy(Oracle)
     assert oracle.governance() == a[0]
     oracle.setGovernance(gov)
     assert oracle.governance() == gov
 
 
-def test_whitelist(gov, oracle):
+@pytest.mark.parametrize("Oracle", ORACLES)
```

Comment:
`ORACLES`?

Reply:
This will run the test case `len(ORACLES)` times, once for each element in `ORACLES` (substituting `Oracle`)